buildpacks/ruby/src/layers/bundle_download_layer.rs

```diff
@@ -113,14 +106,14 @@ impl Layer for BundleDownloadLayer {
             version: self.version.clone(),
         };
         match cache_state(old.clone(), now) {
-            State::NothingChanged(version) => {
-                user::log_info(format!("Using bundler {version} from cache"));
+            State::NothingChanged(_version) => {
+                self.build_output.say("Using cached version");
```

Comment:
I'm not a huge fan of `say` here. Why not `log` or `info`?

Reply:
One intention is trying to distinguish between "logging" which universally everyone (not on this team) assumes means something that the user will never see versus "emit to build output" which is clear that it's intended for the end user. To that end, I'm trying to convey "you are communicating with the user now".

- `log` I want to avoid implying this is a non-visible log
- `info` makes it log-like and makes it seem like there's also `debug` or `fatal` and that that level can be configured. I find that style of "log level" confusing and want to avoid introducing it.

Brainstorming a bit. We could call it:

- out
- output
- println (want to avoid so people don't accidentally use `println!` by mistake)
- emit
- report
- msg
- message
- note (though possibly confusing as they might assume it means `Note: {text}` will be output
- write
- line
- bulletpoint

Or maybe a synonym https://www.thesaurus.com